packages/core/src/codewhisperer/util/supplementalContext/crossFileContextUtil.ts

```diff
@@ -284,14 +284,8 @@ function getSupplementalContextConfig(languageId: vscode.TextDocument['languageI
 
     const group = FeatureConfigProvider.instance.getProjectContextGroup()
```

Comment:
not removing this yet because we might have another ab test coming later for global bm25 sparse retrieval